packages/fields/src/types/Checkbox/views/Controller.js

```diff
@@ -5,8 +5,8 @@ export default class CheckboxController extends FieldController {
     const defaultValue = 'defaultValue' in config ? config.defaultValue : false;
     super({ ...config, defaultValue }, ...args);
   }
-  serialize = data => !!data[this.path];
-  deserialize = data => !!data[this.path];
+  serialize = data => data[this.path];
+  deserialize = data => data[this.path];
```

Comment:
@jordanoverbye I think the `!!` was there so that not-set would serialise to `false` ?

Reply:
This is required otherwise the Admin UI cell will only show `Unchecked` or `Checked`, but never `Not set`.

Reply:
Yep, mases sense